doc/howto/cluster_manage.md

```diff
@@ -5,79 +5,115 @@ discourse: 11330
+```
+
+### Edit the cluster member configuration
 
-As you proceed upgrading the rest of the nodes, they will all
-transition to the Blocked state, until you upgrade the very last
-one. At that point the blocked nodes will notice that there is no
-out-of-date node left and will become operational again.
+To edit all properties of a cluster member, including the member-specific configuration, the member roles, the failure domain and the cluster groups, use the `lxc cluster edit` command.
 
 ## Evacuating and restoring cluster members
 
-Whether it's for routine maintenance like applying system updates requiring
-a reboot or to perform hardware changes, you may sometimes want to empty a
-given server of all its instances.
+There are scenarios where you might need to empty a given cluster member of all its instances (for example, for routine maintenance like applying system updates that require a reboot, or to perform hardware changes).
+
+To do so, use the `lxc cluster evacuate` command.
+This command migrates all instances on the given server, moving them to other cluster members.
+The evacuated cluster member is then transitioned to an "evacuated" state, which prevents the creation of any instances on it.
 
-This can be done using `lxc cluster evacuate <NAME>` which will migrate all
-instances on that server, moving them to other cluster members. The evacuated
-cluster member will be transitioned to an "evacuated" state which will prevent
-the creation of any instances on it.
+You can control how each instance is moved through the [`cluster.evacuate`](instance-configuration) instance configuration key.
+Instances are shut down cleanly, respecting the `boot.host_shutdown_timeout` configuration key.
+
+When the evacuated server is available again, use the `lxc cluster restore` command to move the server back into a normal running state.
+This command also moves the evacuated instances back from the servers that were temporarily holding them.
+
+(cluster-manage-delete-members)=
+## Deleting cluster members
 
-Once maintenance is complete, `lxc cluster restore <NAME>` will move the server
-back into a normal running state and will move its instances back from the servers
-that were temporarily holding them.
+To cleanly delete a member from the cluster, use the `lxc cluster remove` command.
 
-The behavior for a given instance can be configured through the `cluster.evacuate`
-instance configuration key. Instances will be shutdown cleanly, respecting the
-`boot.host_shutdown_timeout` configuration key.
+You can only delete members that are online and that don't have any instances located on them.
+If required, you can force-remove a cluster member (for example, if the server is permanently offline or had to be reinstalled).
+To do so, enter the following command:
+
+    lxc cluster remove --force <member_name>
+
+```{warning}
+Force-removing a cluster member might leave the member in an inconsistent state.
```

Comment:
Good that you added this. I think we should probably go even stronger than this and say that the removed member's database will be left in an inconsistent state (i.e it will certainly have the contents of the cluster DB on it which will relate to thins that are no longer relevant to it) and it is required that the member be wiped and reinstalled.